docker/core/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM anapsix/alpine-java:8 AS build
+COPY codesearch-core.jar / 
+COPY Makefile /
+COPY scripts /scripts/
+VOLUME /data/rust/crates.io-index
+WORKDIR /data/rust/crates.io-index
+RUN git clone https://github.com/rust-lang/crates.io-index
+WORKDIR "/"
+RUN apk update && apk upgrade
+RUN apk add make \
+  git \
+  nodejs \
+  nodejs-npm \
+  go \
+  --no-cache musl-dev \
```

Comment:
What does --no-cache do? and why is it on this line specifically – is there some deeper meaning behind that?

Reply:
I can't remember precisely, but without that command - go wouldn't install. I will find what problem and provide the link.